tests/ui/specialization/min_specialization/bad-const-wf-doesnt-specialize.rs

```
@@ -6,6 +6,7 @@
 struct S<const L: usize>;
 
 impl<const N: i32> Copy for S<N> {}
+//~^ ERROR: `N` is not of type `usize`
```

I don't want to silence errors like this one though.